package.json

```diff
@@ -9,6 +9,7 @@
     "clean": "gatsby clean",
     "author": "LIMIT_BLOG_PAGES=1 SKIP_DOCS=true gatsby develop",
     "build": "gatsby build",
+    "build:incremental": "GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES=true gatsby build",
```

Comment:
Should we avoid building docs? images? (to make it faster)

Reply:
With cached folders `.cache` and `public`, it looks like it's kind of gets skipped i.e only get built if there are any changes. We can consider doing that as well.

Reply:
It'd probably be best to not build images anyway so we don't store a bunch of never-used images in cache nor have to reprocess them all should it ever get cleared.